eventhub_endpoint.bal

```diff
@@ -71,7 +68,7 @@ public client class Client {
         }
         if (publisherId != "") {
             //append publisher ID
-            postResource = postResource + "/publisher/" + publisherId;
+            postResource = postResource + "/publishers/" + publisherId;
         }
         postResource = postResource + "/messages";
         var response = self.clientEndpoint->post(postResource + self.API_PREFIX, req);
```

Comment:
We can use
```http:Response response = <http:Response> check self.clientEndpoint->post(postResource + self.API_PREFIX, request);``` use it if possible in all places.

Reply:
'check' was used wherever possible. Thank you.